src/ray/common/ray_config_def.h

```diff
@@ -63,7 +63,7 @@ RAY_CONFIG(int64_t, worker_lease_timeout_milliseconds, 500)
 /// If set to a negative value, the heartbeats will not be sent.
 /// These are used to report active object IDs for garbage collection and
 /// to ensure that workers go down when the raylet dies unexpectedly.
-RAY_CONFIG(int64_t, worker_heartbeat_timeout_milliseconds, 1000)
```

Comment:
Actually we need these heartbeats even if we aren't sending any IDs to the raylet to ensure workers die when the raylet dies unexpectedly. See my change in https://github.com/ray-project/ray/pull/6412 instead.